var/spack/repos/builtin/packages/redundans/package.py

```diff
@@ -33,12 +36,16 @@ def install(self, spec, prefix):
         filter_file(r'sspacebin = os.path.join(.*)$',
                     'sspacebin = \'' + sspace_location + '\'',
                     'redundans.py')
-        install('redundans.py', prefix.bin)
+
+        binfiles = ['fasta2homozygous.py', 'fasta2split.py',
+                    'fastq2insert_size.py', 'fastq2mates.py',
+                    'fastq2shuffled.py', 'fastq2sspace.py',
+                    'filterReads.py', 'redundans.py']
```

Comment:
What other files are in this directory? Could you use `install_tree`? If there are non-Python files, could you use glob to locate the files to install?

Reply:
There are some skeleton files for dependencies which are provided by depends_on -- including them in the install overrides spack's packages